app/components/Podcasts.vue

```diff
@@ -21,10 +21,14 @@
 import PodcastService from '../api/PodcastService'
 import htmlToText from 'html-to-text'
 import { SET_PLAYER_SCREEN, PLAY_URL, PAUSE } from '../store/constants'
+import config from '../config.js'
 
 export default {
   data: () => {
     return {
+      primaryTextColor: config.colors.primaryText,
```

Comment:
We could acquire some code reading quality if we use object destructuring to the config colors attributes.